openebs/pkg/v1/maya_api.go

```diff
@@ -150,12 +145,12 @@ func (v OpenEBSVolume) ListVsm(vname string, obj interface{}) error {
 		glog.Errorf("HTTP Status error from maya-apiserver: %v\n", http.StatusText(code))
 		return err
 	}
-	glog.Info("VSM Details Successfully Retrieved")
+	glog.Info("Volume Details Successfully Retrieved")
```

Comment:
Can the log statements follow certain standards ? Here everything is Capital cased.
Mention which volume details were retrieved successfully.